lib/many_points.dart

```diff
@@ -0,0 +1,18 @@
+/// The visualise library.
+library many_points;
+
+import 'package:image/image.dart';
+import 'dart:math' as math;
+import 'dart:io' as io;
+
+export 'package:image/image.dart' show Color;
+part 'src/visualisation.dart';
+part 'src/range.dart';
+
+/// Returns [num x, num y, num value]
+typedef List DataTransformFunction(
```

Comment:
I think we might want to be a bit more type restrictive about the return value in the fullness of time. This isn't very discoverable and if people pass in mutable arrays is going to be memory inefficient. Could you add a TODO to explore how the API feels with something like a const class?